### DISCUSSION TITLE
Community Support

### DISCUSSION THREAD

**Discussion by korenwolfke** [zoonk/uneebee#198] [Questions] [answered]
Hello!

I am a developer who has a bit of experience with Elixir, and I came across this project.

I am passionate about Elixir and education/learning, and I am wondering if you are looking for any additional help? I'd love to contribute and help build something awesome if I can.

Appreciate it!

**Comment by ceolinwill** [comment, marked as answer]
Thanks for your interest, @korenwolfje.

The best way to start contributing is by checking out the issues tagged as [good first issue](https://github.com/zoonk/zoonk/labels/good%20first%20issue) or [help needed](https://github.com/zoonk/zoonk/labels/help%20needed). You can also explore [other issues](https://github.com/zoonk/zoonk/issues) that might pique your interest.

Many issues may lack detailed descriptions, so please don’t hesitate to ask for more details or clarifications. If you choose an issue, leave a comment to let me know, and I’ll provide additional specifications to help you get started.

If you’re interested in discussing your potential contributions or have any questions, feel free to reach out.
